conf/machine/qemuriscv64.conf

```diff
@@ -29,3 +29,4 @@ QB_DEFAULT_KERNEL = "bbl"
 QB_NETWORK_DEVICE = "-device virtio-net-device,netdev=net0,mac=@MAC@"
 QB_ROOTFS_OPT = "-drive file=@ROOTFS@,format=raw,id=hd0 -device virtio-blk-device,drive=hd0"
 QB_SLIRP_OPT = "-netdev user,id=net0,hostfwd=tcp::22222-:22"
+QB_OPT_APPEND = "-nographic -serial mon:stdio -serial null"
```

Comment:
Can you explain the reasoning behind this change a bit ?

Reply:
The default args for QEMU will create a display so by adding -nographic we will QEMU to run in text only mode and then after that we just tell QEMU to display the serial output and the QEMU monitor the the users stdio.

It's just being more explicit with the console output.

Reply:
we do have options to control it from runqemu tool
```
runqemu nographic serial 
```
This change sort of overrides the tool behavior, would this
runqemu cmd suffice for your needs ?

Reply:
That would work, but I think that should be the default. Is it possible to make that the default?

Reply:
all qemu targets in OE-Core can boot into graphics hence the default. If we can convince the oe-core community I would suggest to make this change there. I personally have no hard opinion about the default. but I want RISC-V to follow the
oe-core

Reply:
Ok, I've dropped this patch from the PR.